submissions/html/assignment1/2203031080042/aq1.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quotes</title>
+</head>
+<body>
+    <h1>Best Quotes According to Me</h1>
+
+        <blockquote cite="https://www.imdb.com/title/tt0944947/quotes">
+            <p>"Never forget what you are, the rest of the world will not. Wear it like armor and it can never be used to hurt you."</p>
+            <footer>— <cite>Tyrion Lannister</cite></footer>
+        </blockquote>
+        <p>This quote emphasizes <q cite="https://chatgpt.com/">embracing your true self with confidence, turning potential vulnerabilities into strengths that cannot be exploited by others.</q>.</p>
+<hr>
+        <blockquote cite="https://www.imdb.com/title/tt2306299/quotes">
+            <p>"Power is always dangerous. It attracts the worst and corrupts the best."</p>
+            <footer>— <cite>Ragnar Lothbrok</cite></footer>
+        </blockquote>
+        <p>As the saying goes, <q cite="https://chatgpt.com/">the dual nature of power, suggesting it tempts those with ill intent and can morally compromise even those with noble intentions.</q>.</p>
+<hr>
+        <blockquote cite="https://www.imdb.com/title/tt5180504/quotes">
+            <p>"Evil is evil. Lesser, greater, middling… Makes no difference. The degree is arbitrary. The definition’s blurred. If I’m to choose between one evil and another, I’d rather not choose at all."</p>
+            <footer>— <cite>Geralt of Rivia</cite></footer>
+        </blockquote>
+        <p>As the saying goes, <q cite="https://chatgpt.com/">the subjective nature of morality, rejecting the justification of any wrongdoing by degree and advocating for avoiding morally compromising choices altogether.</q>.</p>
+<hr>
+        <blockquote cite="https://www.imdb.com/title/tt5180504/quotes">
+            <p>"No Tree can grow up to Heaven, Unless it’s Roots Reach down to the HELL."</p>
+            <footer>— <cite>Unknown</cite></footer>
+        </blockquote>
+        <p>As the saying goes, <q cite="https://chatgpt.com/">True growth and greatness often require enduring profound struggles and confronting one's deepest challenges.</q>.</p>
+<hr>
+        <blockquote cite="https://www.imdb.com/title/tt5180504/quotes">
+            <p>"When you know how to win you may win many battles, but when you know how to NOT LOOSE you will win each."</p>
+            <footer>— <cite>Jenish Paghadal</cite></footer>
+        </blockquote>

```

Comment:
Redo the PR using a fresh branch! Your work affects the file paths of other's submissions